bin.src/validateCfht.py

```diff
@@ -25,15 +25,12 @@
 import os.path
 import sys
 
-from lsst.validate.drp import checkAstrometry
```

Comment:
Yes!  Integrating and generalizing validateCfht.py and validateDecam.py and having instead a general script that reads a parameter file is DM-4901 and will tackled then.